WordPress/Classes/Networking/AccountSettingsRemote.swift

```diff
@@ -0,0 +1,85 @@
+import Foundation
+
+class AccountSettingsRemote: ServiceRemoteREST {
+    func getSettings(success success: AccountSettings -> Void, failure: ErrorType -> Void) {
+        let endpoint = "me/settings"
+        let path = pathForEndpoint(endpoint, withVersion: ServiceRemoteRESTApiVersion_1_1)
+
+        api.GET(path,
+            parameters: nil,
+            success: {
+                (operation, responseObject) -> Void in
+
+                do {
+                    let settings = try self.settingsFromResponse(responseObject)
+                    success(settings)
+                } catch {
+                    failure(error)
+                }
+            },
+            failure: { (operation, error) -> Void in
+                failure(error)
+        })
+    }
+
+    func updateSetting(change: AccountSettingsChange, success: () -> Void, failure: ErrorType -> Void) {
+        let endpoint = "me/settings"
+        let path = pathForEndpoint(endpoint, withVersion: ServiceRemoteRESTApiVersion_1_1)
+        let parameters = [fieldNameForChange(change): change.stringValue]
+
+        api.POST(path,
+            parameters: parameters,
+            success: {
+                (operation, responseObject) -> Void in
+
+                success()
+            },
+            failure: { (operation, error) -> Void in
+                failure(error)
+        })
+    }
+
+    private func settingsFromResponse(responseObject: AnyObject) throws -> AccountSettings {
+        guard let
+            response = responseObject as? [String: AnyObject],
+            firstName = response["first_name"] as? String,
+            lastName = response["last_name"] as? String,
+            displayName = response["display_name"] as? String,
+            aboutMe = response["description"] as? String,
+            username = response["user_login"] as? String,
+            email = response["user_email"] as? String,
+            primarySiteID = response["primary_site_ID"] as? Int,
```

Comment:
Nice catch. I think when I wrote this there was still no documentation for `/me/settings`. Docs aren't very explicit about it, but I tried on an account with no sites and it returns `0`.

I'm not doing anything with this for profile, but I wanted to get the extra properties in since Account Settings is next